garrysmod/lua/vgui/dvscrollbar.lua

```diff
@@ -274,7 +293,8 @@ function PANEL:PerformLayout()
 	local BtnHeight = Wide
 	if ( self:GetHideButtons() ) then BtnHeight = 0 end
 	local Scroll = self:GetScroll() / self.CanvasSize
-	local BarSize = math.max( self:BarScale() * ( self:GetTall() - ( BtnHeight * 2 ) ), 10 )
+	local BarSize = self:BarScale() * ( self:GetTall() - ( BtnHeight * 2 ) )

```

Comment:
What's the point or doing this change ????

Reply:
It's faster x2

Reply:
It has different output, though maybe the size should not be limited to 10. This is a discussion for another PR though and should not be included in here.

Reply:
> It has different output, though maybe the size should not be limited to 10. This is a discussion for another PR though and should not be included in here.

I disagree